community/advanced_usage/artisan.md

```diff
@@ -0,0 +1,153 @@
+# Artisan CLI
+
+::: tip
+Artisan is a file located within your Pterodactyl installation directory. If you followed the official guide, it should be `/var/www/pterodactyl`.
+:::
+
+::: tip
+To simplify this documentaiton, in command usage you'll see things like the following:
+
+`<hello-world>` - Required argument
+
+`[hello-wolrd]` - Optional argument
+
+`{--hello-world}` - Option
+
+:::
+
+The Artisan CLI, command line interface, is part of the Laravel framework, which Pterodactyl is built on. This guide goes over some more Pterodactyl specific/related Artisan commands, which are all prefixed with the letter `p` (e.g. `p:user:make`). If you'd like to view all commands, you can do so by running:
+
+```bash
+php artisan list
+```
+
+To get information regarding a specific command you can do so by running:
+
+```bash
+php artisan help <command>
+```
+
+## User Management
+
+### Create/Make User
+
+```bash
+php artisan p:user:make {--email=user@example.com}
+                        {--username=myusername}
+                        {--name-first=My}
+                        {--name-last=Name}
+                        {--password=supersecret}
+                        {--admin=1|0}
+                        {--no-password}
+```
+
+If you don't include the optional arguments, it will instead use an interactive prompt to guide you through the user creation process.
+
+### Delete/Remove User
+
+```bash
+php artisan p:user:delete {--user=username/email/UUID}
+```
+
+If you don't include the optional argument, it will instead use an interactive prompt to ask you for the information on the user to remove.
+
+### Disable 2FA
+
+::: warning
+Disabling 2-factor authentication should only be used as a last restort for user recovery. **Please use this with caution**
+:::
+
+```bash
+php artisan p:user:disable2fa {--email=user@example.com}
+```
+
+If you don't include the optional argument, it will instead use an interactive prompt to ask you for the email address associated with the user.
```

Comment:
Since this applies to all user commands, maybe just mention that at the start of the section. The user creation part will also prompt only for the things not passed through, so you can do a combo of args and interaction.